docker/Dockerfile

```diff
@@ -67,8 +67,12 @@ RUN --mount=type=cache,target="/var/cache/apt",sharing=locked \
     apt-get install --no-install-recommends -y git mime-support libxml2 libmariadb3 openssl nodejs && \
     apt-get autoremove -y
 
+# Ensure the npm install is cached, updated, and has a long timeout for slow connections
+# The Node.JS install aboce should install npm 9, but we pin it here just in case
+# npm 10 has shown issues with slow CI environments
```

Comment:
In our installation docs should we recommend users to stay with npm 9 for Nautobot 2.0?

Reply:
I can update that